terminus-ui/select/src/select.component.ts

```diff
@@ -919,73 +793,27 @@ export class TsSelectComponent implements
    */
   public ngOnInit(): void {
     // TODO: re-initialize the selection model if this.allowMultiple changes (rather than throw error like material)
```

Comment:
Do we have a ticket for this?

Reply:
Now there is 😄 https://github.com/GetTerminus/terminus-ui/issues/1528